talent/onboarding/README.md

```diff
@@ -59,6 +59,8 @@ During your first week you are going to have Welcome Talks in order to get to kn
 
 - On Slack we have GeekBot which asks on daily basis a few questions like what you accomplished the previous day, what are you going to work on today and where are you working from. The bot displays your response in the relevant Slack channels. It actually organises asynchronous stand up meetings and keeps our team synced. As people work on different things or/and from different places, it's helps us being transparent and know what everyone is doing. 
 
+You need to configure geekbot here: https://geekbot.com/dashboard/standup/
```

Comment:
Note that Geekbot is usually set up by team leads, since there is a standup configured for each team and members are added/removed there.